src/components/errorPage/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { chakra } from '@chakra-ui/react';
+
+import { translations } from './translations';
+import { StatusCodes } from './statusCodes';
+import Text from '../text';
+import Stack from '../stack';
+import { AutoScout24Logo, MotoScout24Logo } from '../icons';
+import { H1 } from '../heading';
+import Grid from '../grid';
+import Flex from '../flex';
+import Divider from '../divider';
+import Button from '../button';
+import AspectRatio from '../aspectRatio';
+import { Language } from '../../types/language';
+import errorIllustration from '../../assets/images/errorIllustration.png';
+
+interface Props {
+  statusCode: StatusCodes;
+  language: Language;
+  onButtonClick?: () => void;
+}
+
+const ErrorPage: FC<PropsWithChildren<Props>> = ({
+  statusCode,
+  language,
+  onButtonClick,
+  children,
+}) => {
+  const errorTranslations = translations[language][statusCode];
+  return (
+    <Flex height="full" alignItems="center" justifyContent="center">
+      <Stack align="center" spacing="4xl">
+        <Grid columns={2} spacingX="4xl">
+          <AutoScout24Logo width="80px" height="51px" />
+          <MotoScout24Logo width="80px" height="51px" />
+        </Grid>
+        <Divider />
+        <Stack align="center" spacing="2xl">
+          <AspectRatio ratio={4 / 3} maxWidth="400px" width="full">
+            <chakra.img
+              src={errorIllustration}
```

Comment:
How will this work with the CDN/next image setup?

Reply:
rollup converts it to a base64 image and copies it into the build artifact. That is also the reason why it's so compressed. Cannot be too large. This should be a very rare exception where we add images to the components library.